docs/spec.md

```diff
@@ -1853,9 +1861,9 @@ Produces an `output` tensor from a constant `value`.
 
 #### Outputs
 
-| Name     | Type   | Constraints |
-|----------|--------|-------------|
-| `output` | tensor | (C1)        |
+| Name     | Type                                  | Constraints |
+|----------|---------------------------------------|-------------|
+| `output` | tensor or per-tensor quantized tensor | (C1)        |
 
 #### Constraints
 
```

Comment:
`stablehlo.constant` needs special definition for quantized constants.  The value actually holds a `DenseIntElementsAttr` with the storage type, thus the element type will be different.  And the semantic can't be defined as dequant_compute_quant, thus you must specify them here.

Reply:
Thanks @loganchien for the details.
IMO, those details will be pretty helpful in incorporating/implementing  quantized type in StableHLO `constant` op. 

 >> And the semantic can't be defined as `dequant_compute_quant`
I agree. 


>> you must specify them here

In the current PR, the semantics of this op is the input `value` of `constant` type to be interpreter with type as `quantized tensor`. I have updated the constraints of this op to reflect the type constraint. Also updated the syntax of Tensor constants to include constants with quantized type. PTAL.

Also, let me know if I am missing anything.

Reply:
LGTM now.